app/bot.go

```diff
@@ -364,7 +364,7 @@ func (a *App) disableUserBots(userID string) *model.AppError {
 
 		// Get next set of bots if we got the max number of bots
 		if len(userBots) == perPage {
-			options.Page += 1
+			options.Page++
```

Comment:
While this is good to do, but it does not give much gain in readability. Let's revert these changes.

Reply:
Reverting it back to original will cause the lint error. Is it fine?

Reply:
What lint error are you getting for this?